internal/gps/pkgtree/pkgtree.go

```diff
@@ -140,11 +142,19 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 			}
 		} else {
 			switch err.(type) {
-			case gscan.ErrorList, *gscan.Error, *build.NoGoError:
+			case *build.NoGoError:
+				return nil
```

Comment:
unfortunately, we need to preserve this as an error for the general case. we do have a problem in the scanning-our-local-project case with this, yes - but `ListPackages()` serves many callers. it was originally designed for, and is still used by, the solver, which checks if an import statement points to a directory with valid Go code in it. this change would break that check.